tg/controllers.py

```diff
@@ -43,38 +45,166 @@ class Controller:
     # View is terminal vindow
     """
 
-    def __init__(self, model: Model, view: View, tg: Telegram) -> None:
+    def __init__(self, model: Model, view: View, tg: Tdlib) -> None:
         self.model = model
         self.view = view
         self.lock = threading.Lock()
         self.tg = tg
-        self.handlers = {
-            "updateChatDraftMessage": self.update_chat_draft_msg,
-            "updateChatIsMarkedAsUnread": self.update_chat_marked_as_unread,
-            "updateChatIsPinned": self.update_chat_is_pinned,
-            "updateChatLastMessage": self.update_chat_last_msg,
-            "updateChatNotificationSettings": self.update_chat_notification_settings,
-            "updateChatOrder": self.update_chat_order,
-            "updateChatReadInbox": self.update_chat_read_inbox,
-            "updateChatTitle": self.update_chat_title,
-            "updateFile": self.update_file,
-            "updateMessageContent": self.update_msg_content,
-            "updateMessageSendSucceeded": self.update_msg_send_succeeded,
-            "updateNewMessage": self.update_new_msg,
-            "updateMessageContentOpened": self.update_message_content_opened,
-            "updateChatReadOutbox": self.update_chat_read_outbox,
-        }
         self.chat_size = 0.5
         signal(SIGWINCH, self.resize_handler)
 
+        self.chat_bindings: Dict[str, key_bind_handler] = {
+            "q": lambda _: "QUIT",
+            "l": self.handle_msgs,
+            "j": self.next_chat,
+            "^N": self.next_chat,
+            "k": self.prev_chat,
+            "^P": self.prev_chat,
+            "J": lambda _: self.next_chat(10),
+            "K": lambda _: self.prev_chat(10),
+            "gg": self.first_chat,
+            "bp": self.breakpoint,
```

Comment:
:bulb: to get rig of that terrible `_: int` just use lambda `lambda _: self.breakpoint(),`